tests/components/livisi/test_config_flow.py

```diff
@@ -6,76 +6,58 @@
 import pytest
 
 from homeassistant import data_entry_flow
-from homeassistant.components.livisi.const import CONF_HOST, CONF_PASSWORD, DOMAIN
+from homeassistant.components.livisi.const import DOMAIN
 from homeassistant.config_entries import SOURCE_USER
-from homeassistant.const import CONF_NAME
 
-from tests.common import MockConfigEntry
-
-VALID_CONFIG = {
-    CONF_HOST: "1.1.1.1",
-    CONF_PASSWORD: "test",
-}
-
-DEVICE_CONFIG = {CONF_NAME: "foo"}
-
-
-@pytest.fixture(name="config")
-def config_fixture(hass):
-    """Define a config entry data fixture."""
-    return {
-        CONF_HOST: "1.1.1.1",
-        CONF_PASSWORD: "test",
-    }
-
-
-@pytest.fixture(name="config_entry")
-def config_entry_fixture(hass, config):
-    """Define a config entry fixture."""
-    entry = MockConfigEntry(domain=DOMAIN, data=config)
-    entry.add_to_hass(hass)
-    return entry
+from . import VALID_CONFIG, mocked_livisi_controller, mocked_livisi_login
 
 
 async def test_create_entry(hass):
-    """Test that the user step works."""
-    with patch("homeassistant.components.livisi.async_setup_entry", return_value=True):
-        result = await hass.config_entries.flow.async_init(
-            DOMAIN, context={"source": SOURCE_USER}, data=DEVICE_CONFIG
-        )
-
-    assert result["type"] == data_entry_flow.FlowResultType.CREATE_ENTRY
-    assert result["title"] == DEVICE_CONFIG[CONF_NAME]
+    """Test create LIVISI entity."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
 
+    with mocked_livisi_login(), mocked_livisi_controller():
```

Comment:
Please patch the integration setup function on all create entry results to avoid setting up the whole integration. See the scaffold script test template code for example.

Reply:
Here's the example from the scaffold script test code:
https://github.com/home-assistant/core/blob/4f5c9be84f2bbe3cf2f8113b9d44053c2a9c37d4/script/scaffold/templates/config_flow/tests/test_config_flow.py#L22-L25

Reply:
I made the modifications for the tests, please check if they are correct. Thank you.